cli/commands.go

```diff
@@ -258,6 +261,46 @@ func initDaemon(config *conf.MenderConfig,
 	return daemon, nil
 }
 
+func checkDemoCert() {
+	trustStoreCert, err := os.Open(getLocalTrustMenderCertPath())
```

Comment:
You are calling `getLocalTrustMenderCertPath` several times; shouldn't we create a variable and reuse it?